.travis.yml

```diff
@@ -22,14 +22,18 @@ before_install:
 
 install:
   - conda install --yes pip
-  - conda install --yes --file requirements.txt
+  - conda install --yes -c conda-forge --file requirements.txt
+  # We need geopandas master, the legend_kwds is not in 0.3.0
+  - pip install https://github.com/geopandas/geopandas/archive/master.zip
   - pip install libpysal
   # Pysal still needed for code not yet moved to libpysal
   - pip install pysal
   - pip install esda
   - pip install mapclassify
   - pip install -r requirements_dev.txt
-  - pip install .
+  # Now install splot (don't use 'pip install .', we'll run out of space)
```

Comment:
weird... that happens on travis?

Reply:
this was in `/tmp/...`; it seems `pip` copies everything in the folder, which includes the `miniconda` environment.